src/sentry/rules/processing/delayed_processing.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from collections.abc import Mapping
+
+from sentry.buffer.redis import RedisBuffer
+from sentry.models.project import Project
+from sentry.utils.safe import safe_execute
+
+logger = logging.getLogger("sentry.rules.delayed_processing")
+
+
+# TODO: Import once available
+PROJECT_ID_BUFFER_LIST_KEY = "project_id_buffer_list"
+
+
+# TODO: Add redis buffer registry decorator
+def process_delayed_alert_conditions(buffer: RedisBuffer) -> None:
+    project_ids = buffer.get_list(PROJECT_ID_BUFFER_LIST_KEY)  # type: ignore[attr-defined]
```

Comment:
This is `get_set` https://github.com/getsentry/sentry/blob/master/src/sentry/buffer/redis.py#L272-L273